ci-scripts/yocto.groovy

```diff
@@ -135,6 +135,15 @@ void runSmokeTests(String yoctoDir, String imageName) {
     }
 }
 
+void runBitbakeTests(String yoctoDir) {
+    String archiveDir = "bitbake_TestReport"
```

Comment:
You don't actually use this variable. Is it a remnant from the other version of this change?